src/ResolvedPeers.h

```diff
@@ -44,19 +47,20 @@ class ResolvedPeers: public RefCountable
     /// add a candidate path to try after all the existing paths
     void addPath(const Comm::ConnectionPointer &);
 
-    /// re-inserts the previously extracted address into the same position
-    void retryPath(const Comm::ConnectionPointer &);
+    /// makes the previously extracted path available for extraction at its
+    /// original position
+    void returnPath(const PeerConnectionPointer &);
```

Comment:
FTR: The explanation behind this renaming is in the first paragraph of the unofficial commit 46f70173f1ab392bf3b8b6755f0fdfab8124e1e2 message, quoted here:

> I renamed retryPath() because ResolvedPeers do not retry anything. Also, if we keep retryPath(), then PeerConnection::returnable() should become retriable() and that would clash badly with other existing methods and concepts.

Reply:
"return" as a word by itself has multiple primary meanings that imply actions contrary to what you are intending here. IMO "retryPath" was better than this change (while still being bad, I agree on that). For lack of a single better word we may want it to be a descriptive few words instead. "resetUnusedPath()" or somewhat similar?

Reply:
We could try, for example, reAddPath() (symmetrical to the existing addPath()) or reinsertPath().

Reply:
I like reinsertPath()

@rousskov, do you agree on that naming or have a better one?

Reply:
I do not object to reinsertPath() if that is the best we can agree on. I do not like that reinsert implies that it was inserted before while it was actually added (somewhere), not inserted (implying addition "between" other paths).

IMO, reinstatePath() would be better.

Reply:
reinstate sounds good. I'm okay with that or reinsert. Over to you eduard to pick one.

Reply:
Done at ab76848.